acceptance/bundle/validate/sync_patterns/script

```diff
@@ -0,0 +1,4 @@
+trace $CLI bundle validate
```

Comment:
would be good to see json of sync section as well.

Reply:
addressed here https://github.com/databricks/cli/pull/2609/commits/ecb65dcfaec7fbec3562a4412ba43a1098b7fda3